alf/algorithms/ddpg_algorithm.py

```diff
@@ -139,6 +148,17 @@ def __init__(self,
                 gradient dqda element-wise between ``[-dqda_clipping, dqda_clipping]``.
                 Does not perform clipping if ``dqda_clipping == 0``.
             action_l2 (float): weight of squared action l2-norm on actor loss.
+            use_batch_ensemble (bool): whether to use BatchEnsemble FC and Conv2D
```

Comment:
Ideally, we might should make these batch ensemble related parameters transparent to the ddpg_algorithm? Basically, the ddpg_algorithm should not use batch_ensemble related parameters in the ideal case.

Reply:
That's a good point. Currently ddpg needs the ``use_batch_ensemble`` to do some post processing when forwarding critic networks during training. Let me think it over if there might be some alternative methods to work around.